Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:trusty
+
+EXPOSE 443
+
+RUN apt-get update && apt-get -y install python python-setuptools python-virtualenv python-dev \
+  gcc swig dialog libaugeas0 libssl-dev libffi-dev ca-certificates git && \
```

Comment:
Note to self (to be turned into a tracking bug report): We probably ought to create a `bootstrap/ubuntu/trusty.sh` script with all `apt-get`s and call it from travis.yml, call it from Dockerfile and tell people to use it for installation in `using.rst`.